.github/workflows/ci_windows.yaml

```diff
@@ -121,6 +119,27 @@ jobs:
           copy build/qbittorrent.exe upload
           copy build/qbittorrent.pdb upload
           copy dist/windows/qt.conf upload
+          # runtimes
+          copy "${{ env.Qt5_DIR }}/bin/Qt5Core.dll" upload
```

Comment:
why not use windeployqt?

Reply:
I'm not familiar with it, last time I tried I couldn't get it working.
I'll appreciate someone submit a follow up PR that would simplify code.